one_fm/operations/doctype/post_scheduler_checker/post_scheduler_checker.py

```diff
@@ -21,6 +21,7 @@ def validate(self):
 			self.check_date = getdate()
 		self.fill_items()
 		self.get_supervisor()
```

Comment:
@AbrahamAdekunle method name can be changed from get_supervisor to get_shift_supervisor.